client/lib/create-selector/README.md

```diff
@@ -7,11 +7,11 @@ This module exports a single function which creates a memoized state selector fo
 
 `createSelector` accepts the following arguments:
 
-- A function which calculates the cached result given a state object and any number of variable arguments necessary to calculate the result
-- A function or array of functions which return array of dependent state, given the state and the same arguments as the selector
+- A function which selects data from the state. Given a state object and any number of other arguments it calculates a result, which is cached for later reuse.
+- A function which returns the part or parts of the state tree the selector depends on. This can be a single value or an array.
```

Comment:
Slight nit pick here, and it may just be that my writing style is wrong, but I'd add some commas here like so:
"A function which returns the part, or parts, of the state".
I know that this sort of usage works for [appositives](https://data.grammarbook.com/blog/commas/commas-with-appositives/) but I think it still applies when giving alternatives and certainly helps with cadence. Happy to be proven wrong by somebody with better gramma knowledge than myself though!

I think "This can be a single value or an array." could be a little more explicit also.
Does this mean it returns a single value or array of values? 
Does this mean it can be a single function or an array of of functions?

I think it's the later right?
It's definitely a tricky one to explain properly.